<launch>
	<arg name="uav_id" default="1"/>
    <arg name="uav_name" default="uav" />
    
    <!-- 是否运行时自动起飞 -->
    <arg name="auto_takeoff" default="true"/>
    
    <!-- 降落相关参数 -->
    <arg name="land_fixed_height" default="1.2"/>  <!-- 固定高度追踪的高度 -->
    <arg name="land_height_min" default="0.55"/>   <!-- 最低降落高度 -->
    
    <!-- 距离阈值参数 -->
    <arg name="xy_dis_threshold1" default="0.25"/>  <!-- 进入降落程序的水平距离阈值 -->
    <arg name="xy_dis_threshold2" default="0.2"/>  <!-- 触发上锁的水平距离阈值 -->
    
	<node pkg="sunray_tutorial" type="land_on_a_moving_car" name="land_on_a_moving_car_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
		<param name="auto_takeoff" value="$(arg auto_takeoff)" />
		<param name="land_fixed_height" value="$(arg land_fixed_height)" />
		<param name="land_height_min" value="$(arg land_height_min)" />
		<param name="xy_dis_threshold1" value="$(arg xy_dis_threshold1)" />
		<param name="xy_dis_threshold2" value="$(arg xy_dis_threshold2)" />
	</node>
</launch>

